app/Commands/GenerateCommand.php

```diff
@@ -18,8 +18,8 @@ class GenerateCommand extends Command
                             {--force : Overwrite existing files}
                             {--skip : Keep existing files}
                             {--dev : Include dev dependencies like the local .env file}
-                            {--laravel-version= : Set the laravel version}
```

Comment:
Is the laravel-version not needed anymore?

Reply:
oh! you're right! It's not as it's automatically detected.